src/mentionui.js

```diff
@@ -61,6 +70,55 @@ export default class MentionUI extends Plugin {
 		this._mentionsConfigurations = new Map();
 
 		editor.config.define( 'mention', { feeds: [] } );
+
+		this._requestFeedDebounced = debounce( ( marker, feedText ) => {
```

Comment:
I found this bit of code slightly too hard to read due to a long callback for `debounce()`. I'd either move it outside or shortened the code inside it.